docs/source/linchpincli_linchpin_topology.rst

```diff
@@ -1,26 +0,0 @@
-Toplogy
```

Comment:
Can we just comment this out for now? This functionality will return in the near future.